live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml

```diff
@@ -27,8 +27,8 @@
       - curl
       - delphix-go
       - delphix-rust
+      - dh-dkms
```

Comment:
Are we using dkms for the ZFS build?

Reply:
idk why this was added.. don't think so.. lets land as-is, and I can try to follow up and remove it, or determine why it's required.. guessing palash added it for a reason, but not sure why..